public/docs/_examples/dependency-injection/dart/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: dependency_injection
+version: 0.0.1
```

Comment:
Add a description above the version. Maybe:

description: Dependency injection sample